src/rail/estimation/algos/deepdisc.py

```diff
@@ -167,44 +199,46 @@ def run(self):
         # optimizer = return_optimizer(cfg)
         optimizer = solver.build_optimizer(cfg_loader, model)
 
-        """
-        When using the single test dictionary, add this code and replace "mapper" below
-        
-        
+
+        #When using the single test dictionary, add this code and replace "mapper" below
         def dc2_key_mapper(dataset_dict):
             filename = dataset_dict["filename"]
             return filename
 
         IR = DC2ImageReader()
         mapper = RedshiftDictMapper(IR, dc2_key_mapper).map_data
```

Comment:
I think we can modify this assignment to be:
`mapper = RedshiftDictMapper(DCImageReader(), dc2_key_mapper).map_data`

Of course, the `dc2_key_mapper` part will be different if we use a lambda function.

Reply:
#20